.gitignore

```diff
@@ -0,0 +1,6 @@
+api/.DS_store
```

Comment:
You can just add `.DS_Store` without the full path and it will ignore that file in all subdirectories. Similarly, you don't need to fully qualify `ui/node_modules`, you can just do `node_modules`.